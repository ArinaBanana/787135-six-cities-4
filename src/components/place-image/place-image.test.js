import React from "react";
import renderer from "react-test-renderer";
import PlaceImage from "./place-image.jsx";

it(`Should render Place image`, () => {
  const tree = renderer.create(<PlaceImage path={`path`} />).toJSON();

  expect(tree).toMatchSnapshot();
});
