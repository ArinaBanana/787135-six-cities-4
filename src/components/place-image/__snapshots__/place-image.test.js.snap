// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should render Place image 1`] = `
<div
  className="property__image-wrapper"
>
  <img
    alt="Photo studio"
    className="property__image"
    src="path"
  />
</div>
`;
